Makefile

```diff
@@ -24,7 +27,6 @@ check-license:
 	./scripts/check_license.sh
 
 container:
```

Comment:
If promu cross-build is idempotent that we could add a dependency to the crossbuild target for convenience / easier CI integration